clang/include/clang/Serialization/ASTReader.h

```diff
@@ -1205,6 +1205,12 @@ class ASTReader
   /// been completed.
   std::deque<PendingDeclContextInfo> PendingDeclContextInfos;
 
+  struct PendingPreferredNameAttribute {
+    uint64_t RecordIdx;
+    Decl *D;
```

Comment:
Could we use a more descriptive name and/or add a comment explaining why we have this `Decl` here?
`D` looks fine for parameters, but a little too short for a `struct`.